patches/tModLoader/Terraria/Localization/Content/fr-FR/tModLoader.json

```diff
@@ -29,18 +29,18 @@
 		"ModsModPacks": "Packs de Mods",
 		"ModsDisable": "Désactiver",
 		"ModsEnable": "Activer",
-		// "ModConfiguration": "Mod Configuration",
-		// "ModsOpenConfig": "Click to open config",
+		// "ModConfiguration": "Configuration du Mod",
+		// "ModsOpenConfig": "Cliquez pour ouvrir la configuration",
 		"ModOldWarning": "Mod obsolète, activer à vos risques et périls",
 		"ModDependencyClickTooltip": "Ce Mod dépend de : {0}\n (Cliquer pour activer)",
 		"ModDependencyModsNotFound": "Ces Mods n'ont pas été trouvés: {0}",
-		// "ModDidNotFullyUnloadWarning": "This mod did not fully unload during last unload.\n(This is a message for the mod creator)",
-		"BetaModCantPublish": "Ce Mod a été compilé en version Beta et ne peut pas être publié",
+		// "ModDidNotFullyUnloadWarning": "Ce mod ne s'est pas complètement déchargé lors du dernier déchargement.\n(Ceci est un message pour le créateur du mod)",
+		"BetaModCantPublish": "Ce mod a été construit sur la version bêta et ne peut pas être publié",
```

Comment:
The old version of "BetaModCantPublish" was very nice to me, with correct uppercasing of Mod and "Beta" should stay "Beta". "bêta" is someone stupid in French